fastlane/Fastfile

```diff
@@ -78,6 +79,71 @@ lane :release do |options|
   github_release(version: version_number)
 end
 
+desc "Creates GitHub snapshot pre-release and publishes with the 'snapshot' tag"
+lane :snapshot_release do
+  stable_version   = current_version_number
+  snapshot_version = "#{stable_version}-SNAPSHOT.#{Time.now.strftime('%Y%m%d%H%M')}"
+
+  UI.message("Creating snapshot release #{snapshot_version}...")
+
+  Dir.chdir(get_root_folder) do
+    sh("npm version #{snapshot_version} --no-git-tag-version")
+    sh("npm publish --tag snapshot")
```

Comment:
I'm wondering if it would be better to change the library for these snapshots, so we keep the main library clean of all these versions, instead of changing the tag... Mostly so npm stays clean of having a ton of snapshots... But I think it's ok to go with this for now and we can always move to a separate repo in the future, so not a blocker.